src/data/calendar.js

```diff
@@ -0,0 +1,136 @@
+/* global getBoxingDays, getFrenchBankHolidays */
+(function(exports) {
+  "use strict";
+
+  function guessTypeFromComment(comment) {
+    var KNOWN_TYPES = ['RTT', 'CP', 'CS'];
```

Comment:
> I don't think it is a problem to have false positive because you will be able to change it yourself.

Indeed. I take back my previous statement :)